tests/components/keenetic_ndms2/test_config_flow.py

```diff
@@ -86,7 +87,7 @@ async def test_import_works(hass: HomeAssistant, connect):
     assert len(mock_setup_entry.mock_calls) == 1
 
 
-async def test_options(hass):
+async def test_options(hass: HomeAssistant):
```

Comment:
Please type the signature completely when typing.
```suggestion
async def test_options(hass: HomeAssistant) -> None:
```